tabled/src/settings/width/wrap.rs

```diff
@@ -1431,7 +1441,7 @@ mod tests {
                 "\u{1b}[37m(l\u{1b}[39m\u{1b}[37m\u{1b}[41marg\u{1b}[39m\u{1b}[49m\u{1b}[37mest \u{1b}[39m \n",
                 "\u{1b}[37minteger \u{1b}[39m  \n",
                 "\u{1b}[37mless than \u{1b}[39m\n",
-                "\u{1b}[37more equal \u{1b}[39m \n",
+                "\u{1b}[37mor equal \u{1b}[39m \n",
```

Comment:
that's my fault, sorry guys

Reply:
I've re-reviewed my previous `fix typos` `MR` to make sure I didn't make other mistakes and I've reverted those error

At the same time, I fix newly found typos

I hope you pardon my mistake and appreciate the `MR`

#416